README.md

```diff
@@ -36,28 +36,39 @@ Edit `mix.exs` and add `roger` to your list of dependencies and applications:
 
 ```elixir
 def deps do
-  [{:roger, "~> 1.2"}]
+  [{:roger, "~> 3.0"}]
 end
 ```
 
 Then run `mix deps.get`.
 
-### 3. Basic configuration Roger
-
-Configure hosts and default queues:
-
+### 3. Basic configuration amqp
 ```elixir
-config :roger,
-  amqp: [
-    host: "localhost",
-    port: 5672
+config :amqp,
+  connections: [
+    roger_conn: [
```

Comment:
This looks like the connection for rabbitmq. Is `rabbitmq_conn` may be a better name?